src/Resources/config/graphql/Field.types.yaml

```diff
@@ -36,6 +36,9 @@ ImageFieldValue:
             height:
                 type: "Int"
                 resolve: "@=value.height"
+            additionalData:
```

Comment:
What about calling it `focalPoint` ?

```
image {
  focalPoint {
    x
    y
  }
}
```

Reply:
@bdunogier shouldn't it be on par with what we have in the Image fieldtype?

Reply:
Are focalPointX and focalPointY are available in ibexa/oss ?

Reply:
@adamwojs they are not, however, querying them will just return `null`s like shown below:
![image](https://user-images.githubusercontent.com/22300504/210258120-b3fc20b3-2d39-4026-8732-5307758a2f0d.png)

Reply:
OK, fine :)